src/BerconCommon.h

```diff
@@ -139,208 +163,239 @@ enum {
 		p_end,
-	xyz_angle_y, _M("xyz_angle_y"), TYPE_FLOAT, P_ANIMATABLE, IDS_XYZ_ANGLE_Y,
+		xyz_angle_y, _T("xyz_angle_y"), TYPE_FLOAT, P_ANIMATABLE, IDS_XYZ_ANGLE_Y,
 		p_default, 0.f, p_range, -1000000.0f, 1000000.0f,
 		p_ui, TYPE_SPINNER, EDITTYPE_FLOAT, IDC_ANG_Y, IDC_ANG_Y_SPIN, SPIN_AUTOSCALE,
 		p_end,
-	xyz_angle_z, _M("xyz_angle_z"), TYPE_FLOAT, P_ANIMATABLE, IDS_XYZ_ANGLE_Z,
+		xyz_angle_z, _T("xyz_angle_z"), TYPE_FLOAT, P_ANIMATABLE, IDS_XYZ_ANGLE_Z,
 		p_default, 0.f, p_range, -1000000.0f, 1000000.0f,
 		p_ui, TYPE_SPINNER, EDITTYPE_FLOAT, IDC_ANG_Z, IDC_ANG_Z_SPIN, SPIN_AUTOSCALE,
 		p_end,
 
-	xyz_tile_x, _M("xyz_tile_x"), TYPE_INT, P_ANIMATABLE, IDS_XYZ_TILE_X, p_default, x, p_ui, TYPE_INTLISTBOX, IDC_TIL_X, 0, p_end,
-	xyz_tile_y, _M("xyz_tile_y"), TYPE_INT, P_ANIMATABLE, IDS_XYZ_TILE_X, p_default, y, p_ui, TYPE_INTLISTBOX, IDC_TIL_Y, 0, p_end,
-	xyz_tile_z, _M("xyz_tile_z"), TYPE_INT, P_ANIMATABLE, IDS_XYZ_TILE_X, p_default, z, p_ui, TYPE_INTLISTBOX, IDC_TIL_Z, 0, p_end,
+// 	BUG FIX FEB 2024: TILE_Y and TILE_Z were never referenced
+//	xyz_tile_y, _T("xyz_tile_y"), TYPE_INT, P_ANIMATABLE, IDS_XYZ_TILE_X, p_default, y, p_ui, TYPE_INTLISTBOX, IDC_TIL_Y, 0, p_end,
+//	xyz_tile_z, _T("xyz_tile_z"), TYPE_INT, P_ANIMATABLE, IDS_XYZ_TILE_X, p_default, z, p_ui, TYPE_INTLISTBOX, IDC_TIL_Z, 0, p_end,
 
-	// #################### // Random \\ ####################
+		xyz_tile_x, _T("xyz_tile_x"), TYPE_INT, P_ANIMATABLE, IDS_XYZ_TILE_X,
+		p_default, x, p_range, 0, 4, p_ui, TYPE_INTLISTBOX, IDC_TIL_X, 0,
+		p_end,
+		xyz_tile_y, _T("xyz_tile_y"), TYPE_INT, P_ANIMATABLE, IDS_XYZ_TILE_Y,
+		p_default, y, p_range, 0, 4, p_ui, TYPE_INTLISTBOX, IDC_TIL_Y, 0,
+		p_end,
+		xyz_tile_z, _T("xyz_tile_z"), TYPE_INT, P_ANIMATABLE, IDS_XYZ_TILE_Z,
+		p_default, z, p_range, 0, 4, p_ui, TYPE_INTLISTBOX, IDC_TIL_Z, 0,
+		p_end,
 
-	xyz_seed,		_M("seed"), 		TYPE_INT, 		P_ANIMATABLE, 	IDS_SEED, 
+		// #################### // Random \\ ####################
+
+		xyz_seed, _T("seed"), TYPE_INT, P_ANIMATABLE, IDS_SEED,
 		p_default, 12345, p_range, 1, 99999,
-		p_ui, 			TYPE_SPINNER,	EDITTYPE_INT,	IDC_SEED_EDIT,	IDC_SEED_SPIN, SPIN_AUTOSCALE, 
+		p_ui, TYPE_SPINNER, EDITTYPE_INT, IDC_SEED_EDIT, IDC_SEED_SPIN, SPIN_AUTOSCALE,
+		p_end,
+
+		xyz_rand_mat, _T("randByMat"), TYPE_BOOL, P_ANIMATABLE, IDS_RANDBYMAT,
+		p_default, FALSE,
+		p_ui, TYPE_SINGLECHEKBOX, IDC_RAND_MAT2,
 		p_end,
 
-	xyz_rand_mat,	_M("randByMat"),	TYPE_BOOL,		P_ANIMATABLE, IDS_RANDBYMAT,
-		p_default,		FALSE,
-		p_ui,			TYPE_SINGLECHEKBOX, IDC_RAND_MAT,
+		xyz_rand_obj, _T("randByObj"), TYPE_BOOL, P_ANIMATABLE, IDS_RANDBYOBJ,
+		p_default, TRUE,
+		p_ui, TYPE_SINGLECHEKBOX, IDC_RAND_OBJ,
 		p_end,
 
-	xyz_rand_obj,	_M("randByObj"),	TYPE_BOOL,		P_ANIMATABLE, IDS_RANDBYOBJ,
-		p_default,		TRUE,
-		p_ui,			TYPE_SINGLECHEKBOX, IDC_RAND_OBJ,
+		xyz_rand_par, _T("randByPar"), TYPE_BOOL, P_ANIMATABLE, IDS_RANDBYPAR,
+		p_default, FALSE,
+		p_ui, TYPE_SINGLECHEKBOX, IDC_RAND_PAR,
```

Comment:
Is there a reason for fliping these around?

Reply:
sloppiness :) 
fixed